clustered/client/build.gradle

```diff
@@ -31,4 +31,5 @@ dependencies {
   }
   testCompile "org.terracotta.entities:clustered-map-server:$parent.clusteredMapVersion"
   testCompile "org.terracotta:entity-test-lib:$parent.entityTestLibVersion"
+  testCompile "org.terracotta:passthrough-server:$parent.entityTestLibVersion"
```

Comment:
they are in the same repo (https://github.com/Terracotta-OSS/tc-passthrough-testing). I re-used the same property name